lib/plotting_functions.py

```diff
@@ -714,6 +730,17 @@ def make_polar_plot(wks, case_nickname, base_nickname,
         # set levels for difference plot (with a symmetric color bar):
         levelsdiff = np.linspace(-1*absmaxdif, absmaxdif, 12)
     #End if
+    
+    if "pct_diff_contour_levels" in kwargs:
+        levelspctdiff = kwargs["pct_diff_contour_levels"]  # a list of explicit contour levels
+    elif "pct_diff_contour_range" in kwargs:
+            assert len(kwargs['pct_diff_contour_range']) == 3, "pct_diff_contour_range must have exactly three entries: min, max, step"
+            levelspctdiff = np.arange(*kwargs['pct_diff_contour_range'])
+    else:
+        # set levels for difference plot (with a symmetric color bar):
+        levelspctdiff = np.linspace(-1*absmaxpct, absmaxpct, 15)
+        
+    #print("levelspctdiff=======",levelspctdiff)
```

Comment:
Could you remove this commented out line?

Reply:
Done